tests/Fixer/Basic/BracesFixerTest.php

```diff
@@ -4024,17 +4024,16 @@ public function provideFunctionImportCases()
     }
 
     /**
-     * @dataProvider provideFix70Cases
-     * @requires PHP 7.0
+     * @dataProvider provideFixCases
      */
-    public function testFix70(string $expected, ?string $input = null, array $configuration = []): void
+    public function testFix(string $expected, ?string $input = null, array $configuration = []): void
     {
         $this->fixer->configure($configuration);
 
         $this->doTest($expected, $input);
     }
 
-    public function provideFix70Cases()
+    public function provideFixCases(): iterable
```

Comment:
Can we jam these cases into the relevant named tests above instead of having a random "test fix" at the bottom of the file with some other junk? I'd almost rather keep the 70 in the test name, since at least we're saying this is PHP 7.0 specific syntax we're testing here (totally 👍 for removing the redundant `@requires` though).

Reply:
> we're saying this is PHP 7.0

This is not bringing any new information since lowest supported PHP is 7.1, we don't write everywhere that tests are in PHP, not in Java either.

But mostly - BracesFixer is under heavy refactoring by Julien, I'd rather keep changes small and do not touch other functions if not needed (this only applies to the braces tests).

Reply:
Oh yeah, please let's avoid massive changes in `BracesFixer` and `BracesFixerTest` 🙏😄